rb/selenium-webdriver.gemspec

```diff
@@ -53,6 +53,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'rubyzip', ['>= 1.2.2', '< 3.0']
   s.add_dependency 'websocket', ['~> 1.0']
 
+  s.add_dependency 'curb', ['~> 1.0.5']
```

Comment:
This should still be changed to development dependency. We don't want to force every selenium-webdriver user to install curb.

Reply:
Oh thank you, I thought I updated this to development_dependency, let me fix it